igdiscover/germlinefilter.py

```diff
@@ -60,6 +60,9 @@ def add_arguments(parser):
 		'sequence. Default: %(default)s')
 	arg('--maximum-N', '-N', type=int, metavar='COUNT', default=0,
 		help='Sequences must have at most COUNT "N" bases. Default: %(default)s')
+	arg('--maximum-db-diff', '-x', type=int, metavar='N', default=-1,
```

Comment:
For symmetry, the option should be right after `--minimum-db-diff` and be written in the same way. If you think that you have a better description of the options, change it for both.

Reply:
I moved the option --maximum-db-diff below --minimum-db-diff. Note that now both min and max are exposed as possible settings for the user. We needed maximum, but since minimum was already coded I added it to yaml for completeness. By default both are disabled.